unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Video/DCLVideoTexture.cs

```diff
@@ -66,7 +65,10 @@ public override IEnumerator ApplyChanges(BaseModel newModel)
             //If the scene creates and destroy the component before our renderer has been turned on bad things happen!
             //TODO: Analyze if we can catch this upstream and stop the IEnumerator
             if (isDisposed)
+            {
+                var toDebug = this.model as Model;
```

Comment:
is there any porpoise for that or did you forget to remove it after debugging?

Reply:
It was a mistake, already removed it, ty!.